src/main/webapp/scripts/auth.js

```diff
@@ -124,7 +137,12 @@ function setVendorInformationInModal(vendorInformation) {
   registerButton.innerHTML = 'Update Account'
 
   if (vendorInformation.profilePic) {
-    const profilePic = document.getElementById('profile-picture');
+    /** 
+    * ('business-picture') is important to allow the functionality of
+    * setUploadedImage() in both the registration modal and 
+    * the EditCard.html view
+    */
```

Comment:
Nit: Change the comments from /** ... */ to use // at the beginning of each line. /** ... */ is used for JSDoc.